### PR TITLE
Pam 58 closing form after clicking on invite button

### PR DESCRIPTION
**Pull request by saugat977**
Functionality:

- Admin clicks on invite button
- Modal closes only if the data is valid
- Data gets cleared after closing modal

### CODE REVIEW

File: src/modules/InviteUser/components/InviteUser.vue [hunk 1]
```diff
@@ -52,29 +61,39 @@ export default {
       }
     },
     async handleSubmit() {
-      
-      if (!this.name.value || !this.email.value || !parseInt(this.role_id.value)) {
+      if (this.submission.isVerified){
```
Comment (asimnp) [inline src/modules/InviteUser/components/InviteUser.vue:64]: What is the purpose of if (this.submission.isVerified) ?
- Is "!" mark missing?

Comment (saugat977) [reply src/modules/InviteUser/components/InviteUser.vue:64]: the  purpose of if (this.submission.isVerified) is to stop reloading the handleSubmit() function after it is already verified

**Review by asimnp — APPROVED** [review]

File: src/modules/InviteUser/components/InviteUser.vue (hunk 1 above)
Comment (saugat977) [reply src/modules/InviteUser/components/InviteUser.vue:64]: the reloading occured on line 86:  document.getElementById("inviteBtn").click(); 
this line recalls the handleSubmit() Function

Comment (saugat977) [reply src/modules/InviteUser/components/InviteUser.vue:64]: the purpose of if (this.submission.isVerified) is to stop reloading the handleSubmit() function after it is already verified

**Pull request merged**